### ISSUE TITLE
movable and resizable columns

### ISSUE THREAD

**Issue by phil294** [phil294/git-log--graph#33]

**Comment by phil294** [comment]
at least branch lines can now be resized. rest is still tbd not super important imo

**Comment by hansu** [comment]
With movable columns you mean change the order of the columns? I don't think this is neccesary...

**Comment by phil294** [comment]
Yes. Definitely low prio, but this extensions is all about customization and this feature wouldn't affect anybody so I'm all for adding it.
